### DISCUSSION TITLE
Map zoom is slow

### DISCUSSION THREAD

**Discussion by PopescuGeorge94** [Mapsui/Mapsui#2170] [General]
Hi, on my phone OnePlus 8t the map zoom is smooth, but on Huawei p20 lite the zoom is slow:) Any idea why?

**Comment by pauldendulk** [comment]
Could you describe it? I guess you are using pinch zoom. Does the map stay behind relative to where your fingers are touching? Or does fetching of the data take more time?

What in the specs could make the difference? Could it be the CPU (or the GPU if using UseGPU = true), could it be the pixel density?

**Reply by PopescuGeorge94** [reply]
Hi, I am using pinch zoom, the map is not relative, when moving the map I can see the lagging, spec: https://m.gsmarena.com/huawei_p20_lite-9098.php phone with lagging and the phone where everything is smooth https://m.gsmarena.com/oneplus_8t-10420.php. I do not use UseGpu=true
